README.md

```diff
@@ -13,22 +17,30 @@ A monorepo written in Go, represents [Keboola Connection](https://www.keboola.co
 - Configurations are represented as JSON files.
 - Transformations are represented as native files, for example SQL, Python, etc.
 - Read more in the [Documentation](https://developers.keboola.com/cli/).
```

Comment:
Toto by bolo mozno dobre aktualizovat, struktura je takato:
```
Keboola as Code
- CLI
- Templates Service

Buffer Service

```
Tj. to overview nema vela spolocneho s `Buffer`.

Reply:
No to je pod nadpisem `CLI`. `Buffer` je popsaný až pod tím. Ale upravím tu strukturu.